addons/common/functions/fnc_nearestTown.sqf

```diff
@@ -0,0 +1,27 @@
+#include "script_component.hpp"
+/*
+ * Author: 3Mydlo3
+ * Function returns nearest map location for given position/unit.
+ *
+ * Arguments:
+ * 0: Unit/position to find nearest map location <UNIT/POSITION>
+ *
+ * Return Value:
+ * 0: Nearest map location <STRING>
```

Comment:
It does not return a string, it's a https://community.bistudio.com/wiki/Location (which is actually more useful)
```suggestion
 * 0: Nearest map location <LOCATION>
```